wire-schema/src/test/java/com/squareup/wire/schema/SchemaTest.kt

```diff
@@ -707,7 +707,7 @@ class SchemaTest {
   }
 
   @Test
-  fun messsageAndExtensionNameCollision() {
+  fun messageAndExtensionNameCollision() {
```

Comment:
This is the only one I agree with. The others don't seem incorrect in any meaningful way.

Reply:
Yes, you are right about this one. And the other three I think I have missed some details(not typo issue. it's lower camel case issue(I have changed the title)). I have commented under each correction please check.